src/pages/Profile.tsx

```diff
@@ -31,13 +33,48 @@ const GeneralDetailsAboutLine = () => {
   )
 }
 
+const LineProfileHeader = ({
+  operator_ref,
+  agency_name,
+  route_short_name,
+  route_long_name,
+}: {
+  operator_ref: number
+  agency_name: string
+  route_short_name: string
+  route_long_name: string
+}) => {
+  const { t } = useTranslation()
```

Comment:
I think it would be cleaner to extract the into  an interface, something like this : 


```js
interface LineProfileHeaderProps{
  operator_ref: number
  agency_name: string
  route_short_name: string
  route_long_name: string
}
```

Then :

```js
const LineProfileHeader = ({
  operator_ref,
  agency_name,
  route_short_name,
  route_long_name,
}: LineProfileHeaderProps
```

Reply:
Updated, thanks!